test-cloud/pom.xml

```diff
@@ -127,6 +132,112 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>keytool-maven-plugin</artifactId>
+          <executions>
+<!-- Generate a self-signed certificate for the broker keystore:
+$ keytool -genkey -alias broker -keyalg RSA -keystore broker.ks -->
+            <execution>
+              <id>AMQ-generateCertBroker</id>
+              <goals>
+                <goal>generateKeyPair</goal>
+              </goals>
+              <phase>package</phase>
+              <configuration>
+                <alias>broker</alias>
+                <keyalg>RSA</keyalg>
+                <keystore>${certificate.dir}/broker.ks</keystore>
+                <dname>CN=*.project.openshiftdomain</dname>
```

Comment:
What about making this value a parameter as it may differ for other OpenShift instances?